client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -29,15 +29,19 @@ function Transfer( props ) {
 				{ translate( 'Transfer Domain' ) }
 			</Header>
 			<VerticalNav>
-				<VerticalNavItem path={
-					paths.domainManagementTransferOut( slug, selectedDomainName )
-				}>
+				<VerticalNavItem path={ paths.domainManagementTransferOut( slug, selectedDomainName ) }>
 					{ translate( 'Transfer to another registrar' ) }
 				</VerticalNavItem>
-					{ ! isAutomatedTransfer && ! isDomainOnly &&
-						<VerticalNavItem path={ paths.domainManagementTransferToAnotherUser( slug, selectedDomainName ) }>
-							{ translate( 'Transfer to another user' ) }
-						</VerticalNavItem> }
+				{ ! isAutomatedTransfer && ! isDomainOnly &&
+					<VerticalNavItem path={ paths.domainManagementTransferToAnotherUser( slug, selectedDomainName ) }>
+						{ translate( 'Transfer to another user' ) }
+					</VerticalNavItem>
+				}
+				{ ! isAutomatedTransfer &&
```

Comment:
Only on cancel I think. I even think you disabled the transfer at some point, no? :laughing: